common/tests/testCrypto.cpp

```diff
@@ -43,6 +41,20 @@ namespace constants = pdo::crypto::constants;
 // Error handling
 namespace Error = pdo::error;
 
+void Log(int level, const char* fmt, ...)
+{
+    char buf[BUFSIZ] = {'\0'};
+    va_list ap;
+    va_start(ap, fmt);
+    vsnprintf_s(buf, BUFSIZ, fmt, ap);
+    va_end(ap);
+#if _UNTRUSTED_
+    pdo::logger::Log((pdo_log_level_t)level, buf);
+#else
+    ocall_Log(level, buf);
+#endif
+}  // Log
+
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Why not just use the functions in log.cpp? You would basically just call pdo::logger::SetLogFunction(ocall_Log) if trusted and pdo:logger::SetLogFunction(pdo::logger::LogStdOut) if untrusted? I think we can just wrap the printf version of the function with the UNTRUSTED wrapper and the rest should work. That is... making a single copy of Log work would be a much better solution than implementing yet another version.

Reply:
done